specification/columns/chargeclassification.md

```diff
@@ -0,0 +1,38 @@
+# Charge Classification

```

Comment:
Recommend changing the name of this column to `Charge Class`.  It's shorter and means the same thing.  I have not scrubbed the entire file to reflect this; I would want to get buy-in first.

Reply:
please see comment above current preferred option is 'Transaction Category'

Reply:
updated to charge class in 29.3.24 please let me know if there are other alternate suggestions